packages/tasks/src/tasks/image-to-image/about.md

```diff
@@ -1,24 +1,23 @@
 ## Use Cases
 
-### Style transfer
-
-One of the most popular use cases of image-to-image is style transfer. Style transfer models can convert a normal photography into a painting in the style of a famous painter.
-
-## Task Variants
-
 ### Image inpainting
 
-Image inpainting is widely used during photography editing to remove unwanted objects, such as poles, wires, or sensor
-dust.
+Image inpainting is widely used during photography editing to remove unwanted objects, such as poles, wires, or sensor dust.
+
+- check out & learn more: [inpainting demo](https://huggingface.co/spaces/diffusers/stable-diffusion-xl-inpainting), [image inpainting with 🧨diffusers](https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/inpaint)
 
 ### Image colorization
 
 Old or black and white images can be brought up to life using an image colorization model.
 
+- check out & learn more: [photo colorization demo](https://huggingface.co/spaces/modelscope/old_photo_restoration)
+
 ### Super Resolution
 
 Super-resolution models increase the resolution of an image, allowing for higher-quality viewing and printing.
 
+- check out & learn more: [super-resolution demo](https://huggingface.co/spaces/radames/Enhance-This-HiDiffusion-SDXL), [image upscaling with 🧨diffusers](https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/upscale#super-resolution)
+
 ## Inference
```

Comment:
For some reason this header will no longer be rendered properly

Reply:
It's like that for me too, but also in main - https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/tasks/image-to-image/about.md